### ISSUE TITLE
No comparison with vault-agent

### ISSUE THREAD

**Issue by zystem** [tuenti/secrets-manager#74]
Please add comparison with vault-agent https://www.vaultproject.io/docs/platform/k8s/injector

**Comment by fcrespofastly** [comment]
yeah I think the comparisons table I made originally it's kind of obsolete right now. I built that just to justify the investment in the tool, by that time, vault-injector was not built. I think this makes sense @eduardogr

**Comment by jlarfors** [comment]
There is also now the [external-secrets operator](https://external-secrets.io/v0.5.6/provider-hashicorp-vault/) that supports Vault (as well as others) and seems to be a bit more popular.
